src/Controller/ListPageRssController.php

```diff
@@ -305,6 +304,35 @@ protected function getItemList(NodeInterface $node, CacheableMetadata $cache_met
     return $result_items;
   }
 
+  /**
+   * Returns the configuration from the node.
+   *
+   * @param \Drupal\node\NodeInterface $node
+   *   The node.
+   *
+   * @return \Drupal\oe_list_pages\ListPageConfiguration
+   *   The configuration.
+   */
+  protected function getConfigurationFromNode(NodeInterface $node): ListPageConfiguration {
+    return ListPageConfiguration::fromEntity($node);
+  }
+
+  /**
+   * Sets the sort onto the configuration.
+   *
+   * @param \Drupal\node\NodeInterface $node
+   *   The node.
+   * @param \Drupal\oe_list_pages\ListPageConfiguration $configuration
+   *   The configuration.
+   */
+  protected function setSort(NodeInterface $node, ListPageConfiguration $configuration): void {
```

Comment:
First argument in fact is not used.
```suggestion
  protected function setSort(ListPageConfiguration $configuration): void {
```